### PR TITLE
Created the FlatList to list all available books

### PR DESCRIPTION
**Pull request by hernan-clich**
## Summary

Used the `FlatList` component to list all provided books. Created a function in `utils` to trim line breaks.

#### Galaxy S (or similar)

flatlist_galaxyS

#### Pixel 2/3 (or similar)

flatlist_pixel3

#### Pixel 3XL (or similar)

flatlist_pixel3xl

## Trello Card

https://trello.com/c/CkWtamTA/9-n-62-lista-de-libros

### CODE REVIEW

File: wbooks/src/app/screens/Library/index.tsx [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
+import { Book } from '@interfaces/books';
+import { trimLineBreaks } from '@utils/stringUtils';
 
 import styles from './styles';
 
 function Library() {
+  const renderBooks: ListRenderItem<Book> = ({ item }) => (
+    <SingleBook bookCover={item.imageUrl} bookTitle={trimLineBreaks(item.title)} bookAuthor={item.author} />
+  );
+  const keyExtractor = (item: Book) => item.id.toString();
```
Comment (sfernandez11) [inline wbooks/src/app/screens/Library/index.tsx]: `const keyExtractor = ({ id }: Book) => String(id);`

File: wbooks/src/app/screens/Library/index.tsx [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
+
   return (
     <View style={styles.container}>
```
Comment (sfernandez11) [inline wbooks/src/app/screens/Library/index.tsx]: Remove this View, you can use `style` or `contentContainerStyle` directly in the `FlatList`

**Review by sfernandez11 — COMMENTED** [review]
Great, just two commens...

File: wbooks/src/app/screens/Library/index.tsx (hunk 1 above)
Comment (hernan-clich) [reply wbooks/src/app/screens/Library/index.tsx]: Refactored

File: wbooks/src/app/screens/Library/index.tsx (hunk 2 above)
Comment (hernan-clich) [reply wbooks/src/app/screens/Library/index.tsx]: Refactored

**Review by sfernandez11 — APPROVED** [review]
🥇

**Pull request merged**